### ISSUE TITLE
User can view status of products uploaded

### ISSUE THREAD

**Issue by stevebrownlee** [careless-gazelles/Bangazon-Site#15]
**Given** a user is authenticated
**When** a user clicks the *My Products* hyperlink in the navigation bar
**Then** the user will be presented with product status for each product uploaded including "current inventory" and "number sold"

**Given** a user is not authenticated
**Then** the *My Products* hyperlink will not be available in the navigation bar

**Comment by krissycaron** [comment]
This will be checking the Orders and the OrderProduct tables to verify quantity minus sold and remainder of left over for that product. ? @megducharme

**Comment by megducharme** [comment]
Yes, this is for the user if they are a seller. "My Products" will show the products they have uploaded to be sold, and their status - inventory and number sold. The only time the inventory is adjusted is when an order is **complete**. Open orders will not affect the product's inventory.
